### PR TITLE
Add fancy output for cli

### LINKED ISSUE #28: Improve Stdout Formatter
Currently, it is just passing the whole code block to the stdout, which is not interesting given that we can have large code samples.

- [x] ~Add syntax highlight~
- [x] Highlight error line with red bg
- [x] Show only lines around the error

### PR DESCRIPTION
**Pull request by malukenho**
We may move it to another repository once we decouple the output formatter.

### CODE REVIEW

**Comment by EHER** [comment]
Should `adhocore/cli-syntax` required instead of required-dev?

**Comment by EHER** [comment]
Not related but maybe `staabm/annotate-pull-request-from-checkstyle` should also required instead of required-dev.

**Comment by EHER** [comment]
Looks really good! Will be awesome to have output as plugins! ❤️ 

image

**Comment by malukenho** [comment]
I've changed it again. So we don't need any external tools (e.g. for highlight).
I will see if we can backport it to older versions as well.

**Comment by malukenho** [comment]
image

File: src/Validator/MultipleValidator.php [hunk 1]
```diff
@@ -38,7 +39,10 @@ public function supportedLanguage(): array
     public function hasViolation(SampleCode $sampleCode): bool
     {
         foreach ($this->validator as $validator) {
-            if ($validator->hasViolation($sampleCode)) {
+            if (
+                $validator->hasViolation($sampleCode)
```
Comment (EHER) [inline src/Validator/MultipleValidator.php]: Shouldn't be inverted?

Comment (EHER) [reply src/Validator/MultipleValidator.php]: Could be handy to have 
```php
if ($validator->supports($sampleCode) && $validator->hasViolation($sampleCode)) {
    return true;
}
```

**Review by EHER — APPROVED** [review]

**Pull request merged**